### PR TITLE
debuggin logic unified

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @inikulin

### CODE REVIEW

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
FPR

**Comment by helen-dikareva** [comment]
FPR

File: src/test-run/commands/utils.js [hunk 1]
```diff
@@ -13,29 +13,18 @@ function isObservationCommand (command) {
...
-           command.type === TYPE.dragToElement ||
-           command.type === TYPE.selectText ||
-           command.type === TYPE.selectTextAreaContent ||
-           command.type === TYPE.selectEditableContent ||
-           command.type === TYPE.pressKey ||
-           command.type === TYPE.navigateTo ||
-           command.type === TYPE.setFilesToUpload ||
-           command.type === TYPE.clearUpload ||
-           command.type === TYPE.assertion;
+export function shouldBreakBeforeCommand (command) {
```
Comment (inikulin) [inline src/test-run/commands/utils.js]: `canSetDebuggerBreakpointBeforeCommand`

File: src/test-run/commands/utils.js [hunk 2]
```diff
@@ -13,29 +13,18 @@ function isObservationCommand (command) {
...
+    return command.type !== TYPE.debug &&
```
Comment (inikulin) [inline src/test-run/commands/utils.js]: Enumerate commands on **which I could set breakpoint**

**Comment by helen-dikareva** [comment]
FPR

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by AlexanderMoskovkin** [comment]
@helen-dikareva please rebase and we can merge it

**Pull request merged**